constants/providersContent.ts

```diff
@@ -33,170 +47,101 @@ interface ProvidersSection {
   /** A set of links to get more information on the section */
   linkset?: NavLink[]
   /** The sub sections inside the section */
-  subSections?: ProvidersSubSection[]
+  subSections?: ProviderObject[]
   /** The image to show if the section doesn't have subsections */
   image?: string
 }
 
-const realHardware: string = 'real-hardware'
+const convertJsonToObjectData = (jsonData: ProviderObject[]) => {
+  const parsedData = jsonData.map((jsonObject) => {
+    return {
+      title: jsonObject.title,
+      image: jsonObject.image,
+      description: jsonObject.description,
+      installation: jsonObject.installation,
+      websiteCta: {
+        label: jsonObject.websiteCta.label,
+        url: jsonObject.websiteCta.url
+      },
+      docsCta: {
+        label: jsonObject.docsCta.label,
+        url: jsonObject.docsCta.url
+      },
+      sourceCta: {
+        label: jsonObject.sourceCta.label,
+        url: jsonObject.sourceCta.url
+      },
+      helloWorldExample: jsonObject.helloWorldExample
+    }
+  })
+
+  return parsedData
+}
```

Comment:
Not sure why do you need this code... JSON is already an object 🤔 . Have you tried with something like:

```ts
const QUANTUM_RUNTIMES: ProvidersSection = {
    ...
    subSections: rawHardwareData as ProviderObject[]
}
```

Reply:
Addressed in https://github.com/Qiskit/qiskit.org/pull/3000/commits/88ac621a65a1abc7d1e8637babbe6b846672e819